flutter-studio/src/io/flutter/module/FlutterDescriptionProvider.java

```diff
@@ -7,7 +7,7 @@
 
 import com.android.tools.idea.npw.module.ModuleDescriptionProvider;
 import com.android.tools.idea.npw.module.ModuleGalleryEntry;
-import com.android.tools.idea.npw.module.NewModuleModel;
+import com.android.tools.idea.npw.model.NewModuleModel;
```

Comment:
This will work in both 3.1 and 3.2?

Reply:
Exactly. Without adding all that machinery to the plugin tool. Which might be a more robust solution, long term.